lusidtools/cocoon/dateorcutlabel.py

```diff
@@ -7,72 +7,202 @@
 from collections import UserString
 
 
+def _process_timestamp(datetime_value: pd.Timestamp):
+    """
+    Processes pandas timestamp convert it to ISO format and parse to string
+
+    Parameters
+    ----------
+    datetime    pd.Timestamp
```

Comment:
Should parameter docstring follow format used elsewhere in the project?

_:param pd.Timestamp datetime_value:_

Reply:
see numpy post on #client-engineering